lib/msal-node/tsconfig.json

```diff
@@ -19,21 +19,25 @@
     "noImplicitThis": true,
     "alwaysStrict": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedParameters": false,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "resolveJsonModule": true,
     "moduleResolution": "node",
     "jsx": "react",
     "esModuleInterop": true,
     "suppressImplicitAnyIndexErrors": true,
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
+    "baseUrl": ".",
+    "paths": {
+        "@azure/msal-common": ["src/msal-common"]
+    }
   },
   "compileOnSave": false,
   "buildOnSave": false,
   "references": [
     {
-      "path": "../msal-common"
+      "path": "src/msal-common"
```

Comment:
We probably should leave this unchanged. This is what makes "go to definition" work between libs.